python/dgl/graphbolt/dataloader.py

```diff
@@ -221,25 +219,23 @@ def __init__(
             )
             for feature_fetcher in feature_fetchers:
                 feature_fetcher.stream = _get_uva_stream()
-            _find_and_wrap_parent(
-                datapipe_graph,
-                datapipe_adjlist,
-                EndMarker,
-                Bufferer,
-                buffer_size=2,
-            )
-            _find_and_wrap_parent(
-                datapipe_graph,
-                datapipe_adjlist,
-                EndMarker,
-                Awaiter,
-            )
+            if len(feature_fetchers) > 0:
+                datapipe_graph = _find_and_wrap_parent(
+                    datapipe_graph,
+                    EndMarker,
+                    Bufferer,
```

Comment:
and why not improve `_find_wrap_parent()` to support appending multiple datapipes in one call?

Reply:
How are we going to pass the arguments to `_find_wrap_parent()` for the different datapipes in a single call?

Reply:
We are not using `_find_and_wrap_parent` anymore.